charts/pulsar/templates/zookeeper-statefulset.yaml

```diff
@@ -134,10 +134,19 @@ spec:
           containerPort: {{ .Values.zookeeper.ports.clientTls }}
         {{- end }}
         env:
-        - name: ZOOKEEPER_SERVERS
-          value:
-            {{- $global := . }}
-            {{ range $i, $e := until (.Values.zookeeper.replicaCount | int) }}{{ if ne $i 0 }},{{ end }}{{ template "pulsar.fullname" $global }}-{{ $global.Values.zookeeper.component }}-{{ printf "%d" $i }}{{ end }}
+         - name: ZOOKEEPER_SERVERS
+        {{- if .Values.zookeeper.configData.ZOOKEEPER_SERVERS }}
```

Comment:
probably add it to value file with an example and comment it out? so user can know how to use this feature.
As from the script change, seems the provided server list need to contain election port and follower port.

Reply:
@rdhabalia - are you able to update the values.yaml file so that this configuration is documented? I plan to release an updated version of the helm chart next week, and it'd be great to include this.

Reply:
yes, I have added sample-example values in `examples/values-zookeeper-aws.yaml`